versioneer.py

```diff
@@ -0,0 +1,2189 @@
+# Version: 0.22
```

Comment:
do we really need this?

Reply:
Yes :P I mean, we could also look for the functions that we actually use for our use case but normally you would add this whole file.